### ISSUE TITLE
Compress command don't give any action.

### ISSUE THREAD

**Issue by Apogeum12** [vercel/pkg#1167]
When added flag `--compress GZip | Brotli` to the pkg line command don't give any results. 
For example 
```sh
pkg . --options max-heap-size=50 --output somefile
``` 
Gived this same output binary size as:
```sh
pkg  --compress * . --options max-heap-size=50 --output somefile
```
or
```sh
pkg . --options max-heap-size=50  --compress * --output somefile
```
where the * is GZIP or Brotli.
From the other side, some any plan to adding lz4 compressing ?

My platform: linux ubuntu 20.04.02
Target: node14-linux-x64

**Comment by davidzwa** [comment]
Try to check pkg version. v5 has compression, v4 does not (or it seems to me after upgrading to v5 that --compress works)

**Comment by Apogeum12** [comment]
Hey, I checked this before :)  
I have 5.1.0v pkg

**Comment by davidzwa** [comment]
I must say I see the same behaviour now. Well, Im just glad it got reduced from 120MB to 73MB but still... wonky CLI flag

**Comment by Apogeum12** [comment]
So that means the compress work independently(default) ? No matter do we put the -C || -compress or not ?

**Comment by AndresGnu** [comment]
The option compress is don't present in the compilend versión. You verify this inspecting the npm package or view the package in your .node_modules

![Captura de pantalla de 2021-05-09 14-50-43](https://user-images.githubusercontent.com/8808619/117585050-f92e0f00-b0d5-11eb-8869-b413a53fd43a.png)

**Comment by erossignon** [comment]
I confirm the issue:

- the  --compress command works while running pkg from withing the cloned reposotry (development environment) but has no effect with the published version... 
-

**Comment by erossignon** [comment]
@Apogeum12 
In the development folder we have 
![image](https://user-images.githubusercontent.com/680220/117847506-af365c00-b282-11eb-9516-f3227d640c27.png)

This leads me to think that the package has been published without being properly built. 

@robertsLando, have we missed running `npm build` before publishing the package ?

**Comment by robertsLando** [comment]
@erossignon I don't know what's the release process. We should ask this to @leerob

**Comment by erossignon** [comment]
Got it!
The compress options is already advertised on the README.md file but not yet available in @5.1.0

![image](https://user-images.githubusercontent.com/680220/117848570-b742cb80-b283-11eb-95e0-2d4b25c26610.png)

It will be in next version ... 
Be patient

**Comment by Apogeum12** [comment]
Thanks everyone for answers. It was strange to me what it displayed on the output of the pkg --help command. Now I know why.


![screen_pkg_help](https://user-images.githubusercontent.com/35242996/117849201-5bc50d80-b284-11eb-88b0-c23aa8733d08.png)

**Comment by davidzwa** [comment]
> Got it!
> The compress options is already advertised on the README.md file but not yet available in @5.1.0
> 
> ![image](https://user-images.githubusercontent.com/680220/117848570-b742cb80-b283-11eb-95e0-2d4b25c26610.png)
> 
> It will be in next version ...
> Be patient

Oh I recognize this from my own work, advertise a feature and not having it released yet xD Good to know tho guys, keep up the work.

**Comment by Apogeum12** [comment]
Unnecessary I read the README.md ... :) 
I close the issue.
